### PR TITLE
Added fonts, images, icons

### PR DESCRIPTION
**Pull request by llennemann**

### CODE REVIEW

File: src/assets/purpleBackground.png
Comment (vchou105) [inline src/assets/purpleBackground.png:1]: Not sure why the background looks like there are crosses on it but we can fix this later if it looks like an issue after implementing?

**Review by vchou105 — CHANGES_REQUESTED** [review]
I think we are missing the schedule page (purple/pink fuchsia galaxy) and speaker page (black space) backgrounds according to the figma.

File: src/assets/purpleBackground.png
Comment (llennemann) [reply src/assets/purpleBackground.png:1]: For some reason some backgrounds have crosses even though the file on my computer doesn't, but we can fix if it's an issue for sure!

File: src/assets/fonts/futura/futur.ttf
[the remark this answers is not in the document]
Comment (llennemann) [reply src/assets/fonts/futura/futur.ttf:1]: Font package has it like that - not too sure if it's intentional or not

**Review by vchou105 — APPROVED** [review]

**Pull request merged**